data/presets/man_made/water_works.json

```
@@ -12,6 +12,10 @@
         "point",
         "area"
     ],
+    "terms": [
+        "Potable Water Treatment Plant",
+        "Water Treatment Plant"
+    ],
```

```suggestion
    "terms": [
        "water treatment plant"
    ],
```

Lets keep the "potable" out of here; its not mentioned https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dwater_works or https://wiki.openstreetmap.org/wiki/Item:Q6468

And lower case terms, AFAIR.